src/ExpressMiddlewares.js

```diff
@@ -15,21 +15,20 @@ class ExpressMiddlewares {
         const hostName = os.hostname();
         const serviceColor = process.env.SERVICE_COLOR || "unknown";
         const errorHandler = (request, response) => "error";
-        let optKeys = [];
-        let optKeysString = '';
+        const optKeys = [];
 
         // Check for additional access logs
-        if (opts && typeof opts === 'object') {
-          Object.keys(opts).forEach((key) => {
+        if (opts && typeof opts === "object") {
+          for (const key in opts) {
+            console.log(key);
```

Comment:
leaving this in the code, would be a bad idea :fire:

Reply:
I guess it was already removed before you commented on my very last commit. Or should I roll back and commit a new one?